packages/x-tailwindcss/src/x-tailwind-plugin/components/sliding-panel/buttons.ts

```diff
@@ -0,0 +1,73 @@
+/**
+ * Returns the button position variants for the component `sliding panel`.
+ *
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+export function slidingPanelButtons() {
+  return {
+    '.sliding-panel-button': {
+      pointerEvents: 'none',
+      position: 'absolute',
+      transition: 'all ease-out 0.2s',
+      zIndex: 2,
```

Comment:
Would a `zIndex: 1` be enough here considering that the `sliding-panel` has it set to 0?